lib/iex/lib/iex/autocomplete.ex

```diff
@@ -263,6 +312,18 @@ defmodule IEx.Autocomplete do
   defp valid_alias_rest?(<<>>), do: true
   defp valid_alias_rest?(rest), do: valid_alias_piece?(rest)
 
+  ## Elixir Types
+
+  defp expand_elixir_module_types(mod, "", server) do
+    types = match_module_funs(get_module_types(mod), "")
+    format_expansion(types, "")
+  end
+
+  defp expand_elixir_module_types(mod, hint, server) do
+    types = match_module_funs(get_module_types(mod), hint)
+    format_expansion(types, hint)
+  end
```

Comment:
@josevalim A quick question, these 2 functions do the same thing right? Is it necessary to explicitly pattern match the case where `hint` is `""`?